.github/workflows/windows.yml

```diff
@@ -96,8 +96,6 @@ jobs:
 
       - name: Bundle dependencies
         run: |
-          echo "Listing shared library dependencies."
-          ldd "./build/${{matrix.build_type}}/rawtherapee.exe"
```

Comment:
Restore this too before merging

Reply:
I think you have made this change.

Thank you

Jacques